Userland/Applications/Calendar/EventManager.cpp

```diff
@@ -24,12 +24,14 @@ OwnPtr<EventManager> EventManager::create()
 void EventManager::add_event(Event event)
 {
     m_events.append(move(event));
+    m_dirty = true;
```

Comment:
You removed this dirty state a couple of commits ago. Please drop that earlier commit, instead of removing it and then adding it again later.